### DISCUSSION TITLE
minio cluster start error：Unable to use the drive http://xx.xx.xx.xx:9000/opt/data/minio: drive not found

### DISCUSSION THREAD

**Discussion by z624023251** [minio/minio#20734] [Q&A]
OS Version：
![image](https://github.com/user-attachments/assets/9a6f4381-5269-4e31-9b3b-5e870e2a5bc5)

minio version：RELEASE.2024-11-07T00-52-20Z

![image](https://github.com/user-attachments/assets/cb125c04-b073-408b-a3a7-fe37fe21d320)

journalctl log：
![image](https://github.com/user-attachments/assets/7460a1f0-a717-4b3d-82a6-d0e78a413d31)

Single-Node Single-Drive start success，this is the same directory：/opt/data/minio

**Comment by marktheunissen** [comment]
Hi @z624023251 please follow our documentation https://blog.min.io/configuring-minio-with-systemd/ and https://min.io/docs/minio/linux/operations/install-deploy-manage/deploy-minio-multi-node-multi-drive.html

There is not enough information here for us to help and you're just doing something basic wrong, are you using a really old version of systemd that does not support ProtectProc? 

Update your OS in that case

**Reply by z624023251** [reply]
OS Version：
![image](https://github.com/user-attachments/assets/e96cd6c4-36dd-4115-b6eb-1a98965ac715)

**Reply by marktheunissen** [reply]
What do you see in the systemd logs? 

Paste everything from `journalctl -lu minio.service` as well as `cat /etc/default/minio` and `cat /usr/lib/systemd/system/minio.service` and don't blur out the hostnames in screenshots because that could be the issue.

**Reply by marktheunissen** [reply]
What drive are you trying to use? How many hosts are you trying to start? Paste the output from `mount`
